pkg/cri/sbserver/podsandbox/sandbox_delete.go

```diff
@@ -59,10 +57,5 @@ func (c *Controller) Shutdown(ctx context.Context, sandboxID string) error {
 		}
 	}
 
-	// Send CONTAINER_DELETED event with ContainerId equal to SandboxId.
-	c.cri.GenerateAndSendContainerEvent(ctx, sandboxID, sandboxID, runtime.ContainerEventType_CONTAINER_DELETED_EVENT)
-
-	c.sandboxStore.Delete(sandboxID)
```

Comment:
Where is this delete done now?

Reply:
CRI layer is more appropriate place to remove a sandbox from the sandbox store (so it works for all implementations).

We already handle this here: https://github.com/containerd/containerd/pull/8602/files#diff-8782f95b8ffca09f6b312c2018a6c8809364434b93410ebbdab3bacf4cb709e0R105